mmv1/products/compute/FirewallPolicy.yaml

```diff
@@ -0,0 +1,108 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: 'FirewallPolicy'
+api_resource_type_kind: FirewallPolicy
+description: |
+  Hierarchical firewall policy rules let you create and enforce a consistent firewall policy across your organization. Rules can explicitly allow or deny connections or delegate evaluation to lower level policies. Policies can be created within organizations or folders.
+
+  This resource should be generally be used with `google_compute_firewall_policy_association` and `google_compute_firewall_policy_rule`
+
+  For more information see the [official documentation](https://cloud.google.com/vpc/docs/firewall-policies)
+min_version: 'beta'
+references:
+  guides:
+  api: 'https://cloud.google.com/compute/docs/reference/rest/v1/firewallPolicies'
+docs:
+base_url: 'locations/global/firewallPolicies'
+self_link: 'locations/global/firewallPolicies/{{name}}'
+create_url: 'locations/global/firewallPolicies?parentId={{parent}}'
+update_verb: 'PATCH'
```

Comment:
Should this resource be using `legacy_long_form_project`?

Reply:
This resource is not bind to the project.
I tryied to use this `legacy_long_form_project` and the resource fails to create because it remove the reference from the parent.

Eg.: You are using the parent as `organizations/<ORG_ID>`, the code removes the **organizations/** so it will try to use only the **<ORG_ID>** and then throws the following error
```
{
  "error": {
    "code": 404,
    "message": "The resource '<ORG_ID>' was not found",
    "errors": [
      {
        "message": "The resource '<ORG_ID>' was not found",
        "domain": "global",
        "reason": "notFound"
      }
    ]
  }
}
```